docs/faq.rst

```diff
@@ -92,15 +92,9 @@ Common critiques of ``PlainDateTime`` are:
 Are leap seconds supported?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Leap seconds are unsupported.
-Taking leap seconds into account is a complex and niche feature,
-which is not needed for the vast majority of applications.
-This decision is consistent with other modern libraries
-(e.g. NodaTime, Temporal) and standards (RFC 5545, Unix time) which
-do not support leap seconds.
-
-One improvement that is planned: allowing the parsing of leap seconds,
-which are then truncated to 59 seconds.
+Leap seconds are now supported by truncating them away.
+This means that a leap second is treated as if it were the same as the
+preceding second. For example, ``23:59:60`` is treated as ``23:59:59``.
```

Comment:
I'd phrase it like:

> `whenever` does not account for leap seconds. However, it does parse timestamps with a second component of 60, truncating them to 59. This is consistent with other modern datetime libraries.

Reply:
by "account for", I mean: to include actually historical leap seconds in calculations. For example, correctly return that there are 61 seconds between dec 2016 23:59 and jan 2017 00:00.